tests/test_utils.py

```diff
@@ -190,3 +190,46 @@ def test_convert_units():
     truth = r_kpc * (1.0 / d_a) * (180. / np.pi) * 60.
     assert_allclose(utils.convert_units(r_kpc, 'kpc', 'arcmin', redshift, cosmo),
                     truth, **TOLERANCE)
+
+    
+    
+def test_shape_conversion():
+    """ Test the helper function that convert user defined shapes into
+    epsilon ellipticities or reduced shear. Both can be used for the galcat in 
+    the GalaxyCluster object"""
+    
+    def ellipticities(q20,q11,q02):
+        """Build ellipticties from second moments
+        """
```

Comment:
Nitpicky, but can this also include:
`Parameters
    ----------
    q20 : array_like
        Second moment ????
    q11 : array_like
        ????
    q02: array_like
        ????
    Returns
    -------
    x1: array_like
        ??????
    x2: array_like
        ??????
    e1: array_like
        ??????
    e2: array_like
        ??????
`